### PR TITLE
Fix test failures resulting from VSCodes Notebook Start page

### PR DESCRIPTION
**Pull request by DonJayamanne**
Fixes a few of the test failure's we're having, all seem to have the same root cause (hopefully tha'ts the case).

### CODE REVIEW

File: src/client/datascience/notebook/creation/notebookCreator.ts [hunk 1]
```diff
@@ -48,7 +45,6 @@ export class NotebookCreator {
             label: JVSC_EXTENSION_DisplayName
         });
         const placeHolder = DataScience.placeHolderToSelectOptionForNotebookCreation();
-        traceInfoIf(IS_CI_SERVER, `Display quick pick for creation of notebooks ${items.length}`);
```
Comment (DonJayamanne) [inline src/client/datascience/notebook/creation/notebookCreator.ts:51]: Sorry @IanMatthewHuff I thought i removed this in the previous PR, removing here.

File: src/test/datascience/notebook/creation/notebookCreation.vscode.test.ts [hunk 2]
```diff
@@ -33,6 +38,7 @@ suite('DataScience - VSCode Notebook - (Creation Integration)', function () {
         creationOptions = api.serviceContainer.get<CreationOptionService>(CreationOptionService);
         vscodeNotebook = api.serviceContainer.get<IVSCodeNotebook>(IVSCodeNotebook);
         creationOptions.clear();
+        await workAroundVSCodeNotebookStartPages();
```
Comment (DonJayamanne) [inline src/test/datascience/notebook/creation/notebookCreation.vscode.test.ts:41]: Will add this to other tests, once i know which ones are failing.

File: src/client/datascience/notebook/creation/notebookCreator.ts (hunk 1 above)
Comment (IanMatthewHuff) [reply src/client/datascience/notebook/creation/notebookCreator.ts:51]: NBD I filed a code health bug to just clean this usage up all up. No need to sweat it now.

File: src/test/datascience/notebook/helper.ts [hunk 3]
```diff
@@ -416,6 +416,26 @@ export async function stopJupyterServer() {
...
+    if (workedAroundVSCodeNotebookStartPage) {
+        return;
+    }
+    workedAroundVSCodeNotebookStartPage = true;
+    const { editorProvider } = await getServices();
+    await closeActiveWindows();
+
+    // Open a notebook, VS Code will open the start page (wait for 5s for VSCode to react & open it)
+    await editorProvider.createNew();
+    await sleep(5_000);
```
Comment (rchiodo) [inline src/test/datascience/notebook/helper.ts:435]: Blech. Every test takes 5 seconds longer? I guess it's only temporary. 

I'm wondering if there's a way to just wait for the editor open to fire?

**Review by rchiodo — APPROVED** [review]

File: src/test/datascience/notebook/helper.ts (hunk 3 above)
Comment (DonJayamanne) [reply src/test/datascience/notebook/helper.ts:435]: The only way is to see if we first have an active notebook editor, and then we don't have an 
 active notebook editor.
I've filed an issue to remove this hack (will be easier once webview/custom goes away).

Comment (DonJayamanne) [reply src/test/datascience/notebook/helper.ts:435]: FYI https://github.com/microsoft/vscode-jupyter/issues/6113
Yes, all of this sux...

**Review by DavidKutu — APPROVED** [review]

File: src/test/datascience/notebook/helper.ts (hunk 3 above)
Comment (IanMatthewHuff) [reply src/test/datascience/notebook/helper.ts:435]: My hack was to see if we had a visibleEditor, but no active editor, then showNotebookDocument to force it back to showing. That didn't require a wait.

**Pull request merged**